### PR TITLE
Add a Flask template

### PR DESCRIPTION
**Pull request by ajm348**
I'm not really sure if this is how it's supposed to look/be, so feel free to tear this apart

### CODE REVIEW

File: flask/files/.platform/services.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+mysql:
+  type: "mysql:10.1"
+  disk: 2048
+
+redis:
+  type: "redis:3.0"
```
Comment (Crell) [inline flask/files/.platform/services.yaml]: Why not 3.2?

File: flask/files/.platform/services.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+mysql:
+  type: "mysql:10.1"
```
Comment (Crell) [inline flask/files/.platform/services.yaml]: Why not 10.2?

File: flask/files/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Platform.sh Python 2.7 minimal example
```
Comment (Crell) [inline flask/files/README.md]: This file looks like a copy-paste from the minimal Python repo.  It should be updated to say that it's Flask.

File: dodo.py [hunk 4]
```diff
@@ -285,7 +325,7 @@ def common_branch(root):
 def common_push(root):
     return {
         'actions': [
-            'cd %s/template && git checkout update && git push -u origin update' % root,
+            'cd %s/template && git checkout -B update && git push -u origin update' % root,
```
Comment (Crell) [inline dodo.py]: What's the reason for this change?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Utility scripts to manage the Platform.sh Template projects.
```
Comment (Crell) [inline README.md]: Yay, better README!  

No period at the end of titles, though.  Or colons on the others.  Also, leave one blank line between the header and the content below it.

**Comment by Crell** [comment]
Do you not have access to put the branch in the repo directly?  It's harder to collaborate in a fork.

File: dodo.py (hunk 4 above)
Comment (ajm348) [reply dodo.py]: It was because the Flask repo did not have a branch called `update`. Likely unnecessary now that the branch does exist, right? I can remove it.

**Comment by ajm348** [comment]
I don't have access to push to the repo directly.

**Comment by Crell** [comment]
You now have access to push to this repo. :smile:

File: flask/files/README.md (hunk 3 above)
Comment (Crell) [reply flask/files/README.md]: PR every time.  The goal is to hook those PRs up to Psh test environments so we can review the changes before merging them.

File: dodo.py (hunk 4 above)
Comment (Crell) [reply dodo.py]: The branch should be created in the `common_branch()` step.  The push step assumes there's already an update branch to be pushed; if there isn't then adding the `-B` here will make an empty PR, as the new branch will be exactly the same as master.

At least that's how I think it should all work. :smile:

**Comment by pjcdawkins** [comment]
Curious - why python 2.7 and not 3.6?

**Comment by ajm348** [comment]
Ok, I updated versions for redis & mysql, and Crell finally pushed "how it's supposed to work" through my thick skull via slack. Why Python 2.7 vs 3.6... I don't have a good answer for that @pjcdawkins . Personal bias I guess?

**Comment by Crell** [comment]
Thanks, @ajm348!  Merged.
